src/Dynatrace.OpenTelemetry.Exporter.Metrics.Tests/DynatraceMetricsExporterTests.cs

```diff
@@ -662,30 +672,184 @@ public async Task Export_Histogram_CustomBounds()
 			AssertExportRequest(actualRequestMessage);
 		}
 
+		public static IEnumerable<object[]> MinMaxEstimationData =>
```

Comment:
Nice :) 

I have one nit only: The test data is quite long and it kinda gets in the way of reading the tests. We could put in its own class, like the examples here https://andrewlock.net/creating-parameterised-tests-in-xunit-with-inlinedata-classdata-and-memberdata/#loading-data-from-a-method-on-the-test-class

After we are on .NET 6, the new type `HistogramTestData` could even become a record type, that makes it even shorter :)

Reply:
Great idea :slightly_smiling_face: 
Done in [891a873](https://github.com/dynatrace-oss/opentelemetry-metric-dotnet/pull/32/commits/891a873fe94b90877f5d85b4a24b1b0a16b8855b)!